docs/proposals/PROPOSAL_E.md

```diff
@@ -184,6 +177,15 @@ For registries that do not support the `referrers` API, a tag MUST be pushed for
 - Periodic garbage collection may be performed by clients pushing new referrers, deleting stale referrers that have been replaced with newer versions, and tags that no longer point to an accessible manifest.
 - Clients can verify the registry does not support the `referrers` API by querying the API and checking for a 404.
 
+##### Manifest Migration
```

Comment:
"Migration" is going to trigger people. They aren't being moved or altered, just scanned and likely indexed by registries (depending on how they implement the API).